resources/ui/src/main/resources/PhenoTips/SuggestPatientsService.xml

```diff
@@ -61,23 +61,84 @@
   #set ($query = $services.query.xwql($query))
   #set ($discard = $query.setLimit($nb))
   #set ($discard = $query.bindValue('input', "%${input}%"))
-  #set ($discard = $response.setContentType("text/xml"))
-  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
   #set ($results = $query.execute())
-  &lt;results&gt;
+  #if ($request.permission)
+    #set ($requiredPermissions = "$request.permission")
```

Comment:
There's no need to use a string here:

    #set ($requiredPermissions = $request.permission)